onnxruntime/core/providers/cpu/nn/pool.cc

```diff
@@ -235,10 +237,10 @@ Status Pool<float, MaxPool<8 /*VERSION*/>>::Compute(OpKernelContext* context) co
 
   ORT_RETURN_IF_NOT(x_shape.NumDimensions() >= 3, "Input dimension cannot be less than 3.");
 
-  std::vector<int64_t> pads = pads_;
-  std::vector<int64_t> kernel_shape = kernel_shape_;
+  std::vector<int64_t> pads = pool_attrs_.pads;
+  std::vector<int64_t> kernel_shape = pool_attrs_.kernel_shape;
 
-  std::vector<int64_t> output_dims = PoolBase::SetOutputSize(x_shape, x_shape[1], &pads, dilations_, ceil_mode_);
+  std::vector<int64_t> output_dims = pool_attrs_.SetOutputSize(x_shape, x_shape[1], &pads);
```

Comment:
>x_shape, x_shape[1], &pads); [](start = 63, length = 28)

why dilations and ceil_mode are not used in computing output size?

Reply:
dilations and ceil_mode are class members. Seems they don't need to be passed as parameters?